// mpvue-entry/dist/pagesListIndex.js
import Vue from 'vue'
import App from '@/pages/list/index'
import MpvueRouterPatch from 'mpvue-router-patch';
import { log } from 'util';

Vue.use(MpvueRouterPatch);

Vue.prototype.ajax = function (url, data, method) {
  return new Promise((resolve, reject) => {
    wx.request({
     // url: `https://nhjk.uniteddata.com/${url}`,
      url: `http://172.16.1.117:5000/${url}`,
      data: data,
      method: method ? 'POST' : 'GET',
      header: {
        'Authorization': wx.getStorageSync('Authorization'),
        'Content-Type': 'application/x-www-form-urlencoded'
      },
      success(res) {
        console.log(res);
        
        resolve(res.data);
      },
      fail(err) {
        reject(err);
      }
    });
  })
}

Vue.config.productionTip = false



const app = new Vue({
  ...App
})

app.$mount();

export default {
  "config": {
    "navigationBarTitleText": "设备列表",
    "usingComponents": {
      "i-row": "../../iview/row/index",
      "i-col": "../../iview/col/index",
      "i-button": "../../iview/button/index",
      "i-toast": "../../iview/toast/index",
      "i-icon": "../../iview/icon/index",
      "i-modal": "../../iview/modal/index",
      "van-search": "../../vant/search/index",
      "van-swipe-cell": "../../vant/swipe-cell/index"
    }
  }
}
