// mpvue-entry/dist/pagesGatewayEditIndex.js
import Vue from 'vue'
import App from '@/pages/GatewayEdit/index'
import MpvueRouterPatch from 'mpvue-router-patch';

Vue.use(MpvueRouterPatch);

Vue.prototype.ajax = function (url, data, method) {
  return new Promise((resolve, reject) => {
    wx.request({
      // url: `https://nhjk.uniteddata.com/${url}`,
      url: `http://172.16.1.117:5000/${url}`,
      data: data,
      method: method ? 'POST' : 'Get',
      header: {
        'Authorization': wx.getStorageSync('Authorization'),
        'Content-Type': 'application/x-www-form-urlencoded'
      },
      success(res) {
        resolve(res.data);
      },
      fail(err) {
        reject(err);
      }
    });
  })
}

Vue.config.productionTip = false



const app = new Vue({
  ...App
})

app.$mount();

export default {
  "config": {
    "navigationBarTitleText": "添加网关",
    "usingComponents": {
      "i-button": "../../iview/button/index",
      "i-toast": "../../iview/toast/index",
      "van-field": "../../vant/field/index"
    }
  }
}
